corehq/apps/users/views/mobile/users.py

```diff
@@ -756,11 +763,61 @@ def create_mobile_worker(self, in_data):
             phone_number = self.new_mobile_worker_form.cleaned_data['phone_number']
             couch_user.set_default_phone_number(phone_number)
             send_account_confirmation_sms_if_necessary(couch_user)
+
+        data = self.get_plan_and_user_count_by_domain(self.domain)
+        self.check_and_send_limit_email(self.domain, data['plan_limit'], data['user_count'])
         return {
             'success': True,
             'user_id': couch_user.userID,
         }
 
+    @staticmethod
+    def get_plan_and_user_count_by_domain(domain):
+        subscription = Subscription.get_active_subscription_by_domain(domain)
+        plan_version = subscription.plan_version if subscription else DefaultProductPlan.get_default_plan_version()
+        for rate in plan_version.feature_rates.all():
+            if rate.feature.feature_type == 'User':
+                user_rate = rate
+                break
```

Comment:
Adopted the suggestion! and I'm pretty certain all plans do have the User feature type (and so a plan limit) since you can create mobile workers with any plan